iosApp/iosApp/ContentView.swift

```diff
@@ -88,17 +90,70 @@ struct ContentView: View {
         }
     }
 
+    struct LocationAuthButtonStyle: ButtonStyle {
```

Comment:
suggestion (non-blocking): Consider moving this out into a separate LocationAuthButton file